pkg/cloudevents/event_interface.go

```diff
@@ -30,14 +30,17 @@ type EventReader interface {
 	// Extension Attributes
 
 	// Extensions returns the event.Context.GetExtensions().
+	// Extensions use the CloudEvents type system, details in package cloudevents/types.
```

Comment:
In 1.0. extensions are only `map[string]string`

Reply:
I am confused: https://github.com/cloudevents/spec/blob/master/spec.md#extension-context-attributes

so keeping type between binary http back to strong types is going to be very hard.